services/fileServices/MdPageServices/UnlinkedPageService.js

```diff
@@ -0,0 +1,83 @@
+const {
+  retrieveDataFromMarkdown,
+  convertDataToMarkdown,
+} = require("@utils/markdown-utils")
+
+const UNLINKED_PAGES_DIRECTORY_NAME = "pages"
+
+class UnlinkedPageService {
+  constructor({ gitHubService }) {
+    this.gitHubService = gitHubService
+  }
+
+  async create(reqDetails, { fileName, content, frontMatter }) {
+    const newContent = convertDataToMarkdown(frontMatter, content)
+    const { sha } = await this.gitHubService.create(reqDetails, {
+      content: newContent,
+      fileName,
+      directoryName: UNLINKED_PAGES_DIRECTORY_NAME,
+    })
+    return { fileName, content: { frontMatter, pageBody: content }, sha }
```

Comment:
Could we ensure that the third_nav_title is removed from the front matter?

Reply:
Resolved in 1599e5f